client/features/auth/Login.jsx

```diff
@@ -43,13 +43,13 @@ class Login extends React.Component {
     //     redirect: '/'
     //   });
     // }
-    this.props.updateSession('fakehash');
     axios.post('/auth/login', this.state)
       .then((res) => {
         console.log('/login Res', res);
         if (!res.data) {
           alert('Incorrect password');
         } else {
+          this.props.updateSession(this.state.username);
```

Comment:
This is definitely where this should be, but I'm going to ask to temporarily add the earlier line back for testing purposes in my next PR.